test/integration/framework/scheduler_operations.go

```diff
@@ -106,7 +105,7 @@ func parseSchedulerConfiguration(configuration *corev1.ConfigMap) (*config.Sched
 }
 
 // CreateShoot Creates a shoot from a shoot Object
-func (s *SchedulerGardenerTest) CreateShoot(ctx context.Context) (*gardenv1beta1.Shoot, error) {
+func (s *SchedulerGardenerTest) CreateShoot(ctx context.Context) (*gardencorev1alpha1.Shoot, error) {
```

Comment:
we need to set the domain of the cluster to `spec.dns.domain: ""` because currently the example domain `crazy-botany.core.my-custom-domain.com` is used.

Reply:
Now if you do not hand over "external-domain" the Spec.DNS section of the shoot will be cleared. You can hand over an external domain but currently no additional dns providers. I think if we need that in the future, we can easily add it.

Reply:
this is perfect and more than we need today.
Thanks 👍